301d58/code-challenges/filter/challenges-08.test.js

```diff
@@ -0,0 +1,274 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named oddValues that, given an array of integers as input, uses filter to return an array containing only the odd integers.
+
+For example, oddValues([1,2,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const oddValues = (arr) => {
+  return arr.filter ( n => ( n %2 ) );
+
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named filterStringsWithVowels that, given an array of strings as input, uses filter to return an array with only words that contain vowels.
+
+The callback function to filter should include or utilize a regular expression pattern.
+
+For example, filterStringsWithVowels('gregor','hound','xyz') returns ['gregor', 'hound'].
+------------------------------------------------------------------------------------------------ */
+
+
+const filterStringsWithVowels = (arr) => {
+  //Still running into issues with this one.
+  return arr.filter ( value => {if (value === /[aeiou]/) {return true}});
```

Comment:
If you can "ask" if something is equal to a regex, I think it will always return a false because the value isn't actually the same as the regex. When using regex you typically use in it conjunction with a method (.text, .match, .search, etc)